pkg/reconciler/shared/tektonconfig/pipeline/pipeline.go

```diff
@@ -91,6 +91,11 @@ func UpdatePipeline(ctx context.Context, old *v1alpha1.TektonPipeline, new *v1al
 	// if the pipeline spec is changed then update the instance
 	updated := false
 
+	// initialize labels(map) object
+	if old.ObjectMeta.Labels == nil {
```

Comment:
Don't we have annotations in old object ?

Reply:
@savitaashture Seems we do not have `annotations` as well. I do not see an operation on annotation. But we are adding operator version as label.

sample of pipelines CR:
```yaml
apiVersion: operator.tekton.dev/v1alpha1
kind: TektonPipeline
metadata:
  creationTimestamp: "2024-02-12T16:37:54Z"
  finalizers:
  - tektonpipelines.operator.tekton.dev
  generation: 1
  name: pipeline
  ownerReferences:
  - apiVersion: operator.tekton.dev/v1alpha1
    blockOwnerDeletion: true
    controller: true
    kind: TektonConfig
    name: config
    uid: c8e37986-9f52-4d0f-ab5f-eba56415d54c
  resourceVersion: "1086230"
  uid: 9238b012-d29a-4819-94b6-6b6d7ec923eb
spec:
```